RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -250,23 +244,9 @@ extension FactorSourcesClient {
 		try await saveFactorSource(newMainBDFS.embed())
 	}
 
-	public func addOffDeviceFactorSource(
```

Comment:
`OffDeviceMnemonic` is part of MFA - no end user have used this. Might be re-introduced later with Sargon...